news/2018-02-24-Official-Mapping-Contests.md

```diff
@@ -0,0 +1,74 @@
+---
+layout: post 
+title: "Official Mapping Contests: osu! Mapping Olympiad"
+date: 2018-02-24
+--- 
+
+The osu! Mapping Olympiad marks the return of official beatmapping contests. Starting with the osu! gamemode, we're bringing back beatmapping contests in force! Introducing the first contest: Pick one of three songs for your chance to become osu!'s first *Mapping Olympians*!
```

Comment:
for a chance of becoming
Mapping Olympians -> Mapping Olympian?

Reply:
The contest is awarding 3 of such reward title in the form of badges, so I thought Mapping Olympians would be correct here.